src/views/v2/emails/template-parts/body/order/ticket-totals/ticket-price.php

```diff
@@ -21,11 +20,20 @@
  * @var string                             $additional_content The email additional content.
  * @var bool                               $is_tec_active      Whether `The Events Calendar` is active or not.
  * @var \WP_Post                           $order              The order object.
+ * @var array<string, mixed>               $cart_item          The cart item.
  */
 
 use TEC\Tickets\Commerce\Utils\Value;
 
-$ticket_price = Value::create( $ticket->ticket_data['price'] );
+if ( empty( $cart_item['ticket_id'] ) ) {
+	return;
+}
+
+if ( empty( $cart_item['sub_total'] ) ) {
+	return;
+}
+
+$ticket_price = Value::create( $cart_item['sub_total'] );
```

Comment:
I might be wrong, but I find the usage of classes within templates to build data on the fly is something we try to avoid, right?

Reply:
We should, but in this case trying to remove it will take more global changes that are not worth the time now.

Reply:
I agree with Rafsun.